docs/scripting.md

```diff
@@ -22,6 +22,34 @@ module.exports = (robot) ->
 
 The `robot` parameter is an instance of your robot friend. At this point, we can start scripting up some awesomeness.
 
+## Creating your own script
+
+If you have an idea for a script that doesn't yet exist in the [NPM Registry](https://www.npmjs.com/browse/keyword/hubot-scripts) then you can easily get started using the `hubot` script [yeoman](http://yeoman.io/) generator.  This requires that you have the hubot generator installed:
+
+```
+% npm install -g yo generator-hubot
```

Comment:
I'm unsure about having to cut/paste some of the common setups for `generator-hubot`.  I feel like this is okay for now until we collapse the documentation a little more.